### PR TITLE
Add tutorial for creating automatically xlf translation file - Update new-system.md

### PR DESCRIPTION
**Pull request by vcopleutre**
For new module developpers, one of our questions is "how to create the translation file".

This part add the best practice (according to my own judgement) do to it.

The syntax may not be efficient, feel free to reword it or add internal anchors to the text.

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       |  8.x
| Description?  | Answering the question for developpers : "how to create automatically the xlf translation file in my module"
| Fixed ticket? | none

### CODE REVIEW

File: modules/creation/module-translation/new-system.md [hunk 1]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
 Note that these files will only work with PrestaShop 1.7.8 and over. If you need to ensure compatibility with previous versions, read below.
 {{% /notice %}}
 
+### Recommanded practice for creating module translations
+
+For module developpers, if you want to incorpore your final translations into your own module without creating the xlf file by hand, the recommanded practice is to :
```
Comment (thomasnares) [inline modules/creation/module-translation/new-system.md]:
```suggestion
For module developers, if you want to incorporate your final translations into your own module without creating the xlf file by hand, the recommended practice is to :
```

File: modules/creation/module-translation/new-system.md [hunk 2]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
...
+
+1. Prepare your translations (see "Making your wordings appear in the translation interface")
```
Comment (thomasnares) [inline modules/creation/module-translation/new-system.md]:
```suggestion
1. Prepare your translations (see "[Making your wordings appear in the translation interface]({{<relref "#making-your-wordings-appear-in-the-translation-interface">}})")
```

File: modules/creation/module-translation/new-system.md [hunk 3]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
...
+2. Translate them manually in the back office
+3. Export the translations (see "Exporting translations") and add them in your module translations folder
```
Comment (thomasnares) [inline modules/creation/module-translation/new-system.md]:
```suggestion
3. Export the translations (see "[Exporting translations]({{<relref "#exporting-translations">}})") and add them in your module translations folder
```

**Review by thomasnares — DISMISSED** [review]
Thanks @vcopleutre, i added a few suggestions with internal linking 👍

File: modules/creation/module-translation/new-system.md [hunk 4]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
 Note that these files will only work with PrestaShop 1.7.8 and over. If you need to ensure compatibility with previous versions, read below.
 {{% /notice %}}
 
+### Recommended practice for creating module translations
+
+For module developers, if you want to incorporate your final translations into your own module without creating the xlf file by hand, the recommended practice is to :
```
Comment (kpodemski) [inline modules/creation/module-translation/new-system.md]:
```suggestion
For module developers, if you want to incorporate your final translations into your module without creating the xlf file by hand, the recommended practice is to:
```

File: modules/creation/module-translation/new-system.md [hunk 5]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
...
+
+1. Prepare your translations (see "[Making your wordings appear in the translation interface]({{<relref "#making-your-wordings-appear-in-the-translation-interface">}})")
+2. Translate them manually in the back office
```
Comment (kpodemski) [inline modules/creation/module-translation/new-system.md]:
```suggestion
2. Translate them manually in the back office.
```

File: modules/creation/module-translation/new-system.md [hunk 6]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
 Note that these files will only work with PrestaShop 1.7.8 and over. If you need to ensure compatibility with previous versions, read below.
 {{% /notice %}}
 
+### Recommended practice for creating module translations
+
+For module developers, if you want to incorporate your final translations into your own module without creating the xlf file by hand, the recommended practice is to :
+
+1. Prepare your translations (see "[Making your wordings appear in the translation interface]({{<relref "#making-your-wordings-appear-in-the-translation-interface">}})")
```
Comment (kpodemski) [inline modules/creation/module-translation/new-system.md]:
```suggestion
1. Prepare your translations (see "[Making your wordings appear in the translation interface]({{<relref "#making-your-wordings-appear-in-the-translation-interface">}})").
```

File: modules/creation/module-translation/new-system.md [hunk 7]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
...
+2. Translate them manually in the back office
+3. Export the translations (see "[Exporting translations]({{<relref "#exporting-translations">}})") and add them in your module translations folder
```
Comment (kpodemski) [inline modules/creation/module-translation/new-system.md]:
```suggestion
3. Export the translations (see "[Exporting translations]({{<relref "#exporting-translations">}})") and add them to your module translations folder.
```

File: modules/creation/module-translation/new-system.md [hunk 8]
```diff
@@ -341,6 +341,16 @@ You can distribute the downloaded dictionaries by placing the extracted files in
...
+
+These translations will be set up in Prestashop during the installation of the module.
```
Comment (kpodemski) [inline modules/creation/module-translation/new-system.md]:
```suggestion
These translations will be set up in PrestaShop during the module's installation.
```

**Review by kpodemski — COMMENTED** [review]
thanks! a few comments and it's all good :)

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thanks for contributing @vcopleutre !

**Comment by vcopleutre** [comment]
Thank you for the updates!

**Pull request merged**